CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # Changelog
 ## Unreleased
 * **improvement** Changed the name of the framework from `Cauli` to `Cauliframework`. [#135](https://github.com/cauliframework/cauli/issues/135)
-
-## Unreleased
+* **improvement** The ViewControllerShakePresenter is now wrapping the to displaying ViewController in a UINavigationController. [#114](https://github.com/cauliframework/cauli/issues/114)
```

Comment:
I think I wouldn't focus on that, since the `ViewControllerShakePresenter` is internal anyways. I think I would just talk about that we will now show a done button if the Cauli ViewController is displayed via the shake gesture. What do you think?

Reply:
You're right. Since it's internal, it does not need to be that detailed. I'll update it.